.travis.yml

```diff
@@ -13,13 +13,7 @@ jobs:
         - brew install clang-format
         - brew install swiftformat
       script:
-        - ./scripts/check_whitespace.sh
-        - ./scripts/check_copyright.sh
-        - ./scripts/check_no_module_imports.sh
-        - ./scripts/check_test_inclusion.py
-        - ./scripts/style.sh test-only $TRAVIS_COMMIT_RANGE
-        # Google C++ style compliance
-        - ./scripts/lint.sh $TRAVIS_COMMIT_RANGE
+        - ./scripts/check.sh --test-only $TRAVIS_COMMIT_RANGE
```

Comment:
Looks like a problem at https://travis-ci.org/firebase/firebase-ios-sdk/jobs/506855309

If there is a failure in one of these encompassed scripts, will it be clear enough which one?  Should check.sh log that it is starting each one?

Reply:
Yeah, the issue was that parsing a revision range results in an expansion of up to three revisions which git diff doesn't like. It will accept ranges though. I've updated check.sh to pass revision ranges through.